test/system/700-play.bats

```diff
@@ -79,7 +79,7 @@ RELABEL="system_u:object_r:container_file_t:s0"
 
     # Make sure that the K8s pause image isn't pulled but the local podman-pause is built.
     run_podman images
-    run_podman 1 image exists k8s.gcr.io/pause
+    run_podman 1 image exists registry.k8s.io/pause
```

Comment:
I would much rather see this as *both* lines: adding `registry.k8s` right below the `k8s.gcr` line.

Reply:
Just curious about the rationale behind keeping the `k8s.gcr.io` line as well.

Reply:
Yeah, it makes sense. Thanks for the explanation.